pkgs/applications/version-management/git-and-tools/git-team/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "git-team";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "hekmekk";
+    repo = "git-team";
+    rev = "v${version}";
+    sha256 = "0nl5j64b61jw4bkf29y51svjbndmqqrqx96yaip4vjzj2dx9ywm4";
+  };
+
+  vendorSha256 = null;
+
+  proxyVendor = true;
+
```

`proxyVendor` isn't needed.
```suggestion
  vendorSha256 = "sha256-xJMWPDuqoNtCCUnKuUvwlYztyrej1uZttC0NsDvYnXI=";
```

---

How can I obtain this hash? Would come in handy to know for updates to come. I didn't find a good explanation yet.

---

I updated default.nix accordingly and could just move on, but I don't just want to do stuff I don't fully understand. If you can give an explanation or point me into the right direction, that would be great.

---

Not sure I understand what you're asking? https://github.com/NixOS/nixpkgs/pull/164372#discussion_r827926584 explains why the previous approach was incorrect?

---

Well yes, but I still don't fully understand why vendorSha256 is needed to guarantee a reproducible build, because there is also the hash of the revision.🤔
I was hoping for an easy explanation or link why that is.

---

I found some discussion in googlegroups. Fine for me now.